docs/music.rst

```diff
@@ -4,8 +4,10 @@ Music
 .. py:module:: music
 
 This is the ``music`` module. You can use it to play simple tunes, provided
-that you connect a speaker to your board.
+that you connect a speaker to your board.      
+Note that the ``music`` module is no longer included in the ``microbit`` module. To use it, it must be imported separately with::
```

Comment:
I don't think we need to refer to legacy behaviour.  So simply say "Note thate the music module is not included in the ..."